backend/service/chaos/experimentation/experimentstore/experiment_config.go

```diff
@@ -0,0 +1,29 @@
+package experimentstore
+
+import (
+	"github.com/golang/protobuf/ptypes/any"
+
+	experimentation "github.com/lyft/clutch/backend/api/chaos/experimentation/v1"
+)
+
+type ExperimentConfig struct {
+	id     uint64
+	Config *any.Any
+}
+
+func (ec *ExperimentConfig) CreateProperties(transformer *Transformer) ([]*experimentation.Property, error) {
+	properties, err := transformer.CreateProperties(ec)
```

Comment:
create properties with transformers registered by external packages - `serverexperimentation` in our case.